exercises/space-age/lib/example.dart

```diff
@@ -0,0 +1,69 @@
+class SpaceAge {
+  /// This is a computed value so in order to
+  /// prevent any accidental assignments it is
+  /// made private.
+  double _ageInYears = 0.0;
+
+  int _ageInSeconds = 0;
+  Planets choiceOfPlanet = Planets.Earth;
+
+  SpaceAge([int ageInSeconds = 0, this.choiceOfPlanet = Planets.Earth]) {
+    this.inSeconds = ageInSeconds;
+  }
+
+  /// Returns age (`double`) in accordance with [choiceOfPlanet].
+  /// Getter for [_ageInYears].
+  double get age => num.parse(_ageInYears.toStringAsFixed(2));
+
+  /// Expects a planet p which is a value from `enum` Planets.
+  /// Setter for [choiceOfPlanet].
+  set planet(Planets p) {
+    this.choiceOfPlanet = p;
+    this.inSeconds = this._ageInSeconds;
+  }
+
+  /// Getter for [_ageInSeconds].
+  int get inSeconds => this._ageInSeconds;
+
+  /// Setter for [_ageInSeconds].
+  set inSeconds(int ageInSeconds) {
+    this._ageInSeconds = ageInSeconds;
+    int earthYearInSeconds = 31557600;
+
+    switch (choiceOfPlanet) {
+      case Planets.Earth:
+        this._ageInYears = (ageInSeconds / earthYearInSeconds);
+        break;
+      case Planets.Mars:
+        this._ageInYears = (ageInSeconds / earthYearInSeconds) / 1.8808158;
+        break;
+      case Planets.Mercury:
+        this._ageInYears = (ageInSeconds / earthYearInSeconds) / 0.2408467;
+        break;
+      case Planets.Jupiter:
+        this._ageInYears = (ageInSeconds / earthYearInSeconds) / 11.862615;
+        break;
+      case Planets.Neptune:
+        this._ageInYears = (ageInSeconds / earthYearInSeconds) / 164.79132;
+        break;
+      case Planets.Saturn:
+        this._ageInYears = (ageInSeconds / earthYearInSeconds) / 29.447498;
+        break;
+      case Planets.Uranus:
+        this._ageInYears = (ageInSeconds / earthYearInSeconds) / 84.016846;
+        break;
+      case Planets.Venus:
+        this._ageInYears = (ageInSeconds / earthYearInSeconds) / 0.61519726;
+        break;
+    }
+    ;
+  }
+
+  /// Returns `double` rounded of to [fractionDigits].
+  double roundOf(double number, {int fractionDigits = 2}) => num.parse(number.toStringAsFixed(fractionDigits));
```

Comment:
Forgot to actually use this method!

Reply:
Are you going to drop this method then? Or are you going to use it?